services/api/src/messaging/players.service.ts

```diff
@@ -36,6 +37,33 @@ export const findPlayerById = (
   return player;
 };
 
+export const findPlayerByConnection = (ws: GameWebSocket): Player => {
+  const players = gameRoomService.getPlayers(ws.roomId);
+  const player = players.get(ws);
+  if (!player) {
+    throw new Error('player with this ws connection not found');
+  }
+  return player;
+};
+
+export const checkIfLastToVote = (
+  roomId: string,
+  playerId: string,
+): boolean => {
+  const player = findPlayerById(roomId, playerId);
```

Comment:
What is the logic behind puting methods into services?
Why is this method inside the players service, while `getPlayers`  and `getVoterWhoMadeActionCount` are in gameService and gameRoomService?